tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -5783,7 +5783,7 @@ Status ParseEquation(OpConverterParams* params,
   OperandLabelCounts input_label_counts;
   LabelCounts output_label_counts;
   absl::InlinedVector<bool, 2> input_has_ellipsis;
```

Comment:
Could you initialize these too? [ParseEquation](https://github.com/tensorflow/tensorflow/blob/b47357628fa624676db35186d57b416462f8bcd5/tensorflow/core/kernels/linalg/einsum_op_impl.h#L160) sets also those conditionally, and we use them below.

Reply:
These are all inlined vector which are empty as declared here. The template parameter is just a capacity if I understand correctly.